src/components/ItemSelector/styles/ItemSelector.style.js

```diff
@@ -1,4 +1,4 @@
-import { colors } from './colors'
+import { colors } from '../../../modules/colors'
```

Comment:
We should import `colors` from `@dhis2/ui-core` instead

Reply:
Yes indeed, but thought this was out of scope, so I started with just consolidating all the colors in to the same file at least (previously there were 2). Created a separate task for the colors -> ui-core colors: https://jira.dhis2.org/browse/TECH-345